.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: "❓ IRC"
-    url: https://web.libera.chat/
-    about: Please ask and answer questions here, (#jenkins-release on libera.chat).
```

Comment:
While the Matrix link is useful for those who use it or would use webchat, this basically makes the link useless to anyone who's not using it, like the old "click here" antipattern.